conf/test_all.config

```diff
@@ -0,0 +1,35 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/phaseimpute -profile test_sim,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'Test simulation / imputation / validation mode'
+    config_profile_description = 'Minimal test dataset to check pipeline function'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
+
+    // Input data
+    input                = "${projectDir}/tests/csv/sample_sim.csv"
+    input_region         = "${projectDir}/tests/csv/region.csv"
```

Comment:
I've modified the description of the input_region for the moment.

Reply:
I have a question, what about making `--input_region` non mandatory? Then, we can have two options: 

- if the the `region.csv` was provided (and ends in `.csv`, etc) then the `ch_regions` are processed as now. 
- Otherwise, when there is no `region.csv`, `input_region` is assigned to "all" and then the regions are obtained with the fasta (`GET_REGION` subworkflow lines 12-20). 

This would mean that when the user does not provide a `region.csv` then we assume that wants to compute all regions.